test/policies/basic-auth-policy.test.js

```diff
@@ -72,34 +72,24 @@ describe('Functional Tests basic auth Policy', () => {
             should.exist(_fromDbUser1);
             user = _fromDbUser1;
 
-            credentialService.insertScopes('authorizedScope', 'unauthorizedScope')
+            return credentialService.insertScopes('authorizedScope', 'unauthorizedScope')
               .then(() => {
```

Comment:
this should be extracted to the higher level. 
it should be 
f
.then
.then

not f.then(x=>x.then)

current syntax works for callback style, but once you start do `return` it is super easy to miss correct place to return

Reply:
Fixed in 2205b78